pkg/diff/util.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package diff
+
+import (
+	"math/rand"
+
+	"github.com/ngaut/log"
+	"github.com/pingcap/tidb-tools/pkg/dbutil"
+	"github.com/pingcap/tidb/model"
+	"github.com/pingcap/tidb/types"
+)
+
+func equalStrings(str1, str2 []string) bool {
```

Comment:
where use this function? and why not use `==` simply

Reply:
slice can only be compared to nil. this function is used to check whether two schema have same tables, I will use this function later

Reply:
wow, my mistake